test/SourceKit/NameTranslation/basic.swift

```diff
@@ -66,6 +67,7 @@ class MyDerived: FooClassDerived {
 // CHECK9: fooProperty11
 // CHECK10: init(nstanceFunc21:)
 // CHECK11: init(lassDerived2:)
+// CHECK11-1: FooClassDerived2
```

Comment:
Is that something that changed in this commit, because it's not what I see right now.  I see it trying to find the initializer name, which then causes us to change `Foo` to `fooWithX` in the ObjC code.